deps.bzl

```diff
@@ -2977,3 +2977,213 @@ def prysm_deps():
         sum = "h1:4Z09Hglb792X0kfOBBJUPFEyvVfQWrYT/l8h5EKA6JQ=",
         version = "v0.0.0-20190525122527-15d366b2352e",
     )
+    go_repository(
```

Comment:
why are all of geth's deps re-imported here ? we are already using a rule for it

Reply:
Gazelle did it. 

One thing to keep in mind is that this `go_repository` alias is a "maybe" style import. So if we have already imported geth's dependencies via bazel rule, then this would not override it.